### DISCUSSION TITLE
Would it be possible to have the dev server return __typename on objects?

### DISCUSSION THREAD

**Discussion by tomschut** [maoosi/prisma-appsync#113] [Q&A] [answered]
Hi, 

Perhaps a weird question, but our front-end sometimes uses __typename to determine types returned from appsync. Appsync supplies these types, and to have it work locally we'd need them there too. 

So my question is:  would it be possible to have yoga server return the typenames as well?

Kind regards,

Tom

**Comment by maoosi** [comment, marked as answer]
Hey @tomschut,

Interesting use of __typename! And yes, definitely possible to return it from the Yoga server.

I have created a new issue: https://github.com/maoosi/prisma-appsync/issues/115
